package.json

```diff
@@ -113,7 +113,7 @@
     "classlist.js": "1.1.20150312",
     "core-js": "3.7.0",
     "focus-trap": "6.5.0",
-    "fundamental-styles": "0.18.1-rc.25",
+    "fundamental-styles": "0.19.0-rc.18",
```

Comment:
Have you checked if the Breaking changes introduced in 0.19. have been addressed and not breaking anything in Fund-ngx?

Reply:
for release 19 of fundamental-styles, we have 4 component changes as breaking change. That should be addressed as part of a separate PR. 
for this PR changes to work need atleast v19.0.rc.14.
Have to wait till those breaking changes are addressed and new release is done.
marking as blocked for now.